### PR TITLE
SONARHTML-144 Support SalesForce Aura Lightning Components: update default suffixes with ".cmp"

### PR DESCRIPTION
**Pull request by yassin-kammoun-sonarsource**

### CODE REVIEW

File: its/plugin/pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.sonarsource.html</groupId>
     <artifactId>html-its</artifactId>
-    <version>3.4.0-SNAPSHOT</version>
+    <version>3.5.0-SNAPSHOT</version>
```
Comment (roberto-orlandi-sonarsource) [inline its/plugin/pom.xml:8]: was it intended to increase also the plugin version in this PR?

Comment (yassin-kammoun-sonarsource) [reply its/plugin/pom.xml:8]: Oups, I will revert the commit.

Comment (roberto-orlandi-sonarsource) [reply its/plugin/pom.xml]: I will wait for it to be green again and then I'll approve it :)

Comment (nils-werner-sonarsource) [reply its/plugin/pom.xml]: However, I just noticed, that the its/plugin pom version is not identical to the other pom versions. Should be fixed in another commit.

Comment (yassin-kammoun-sonarsource) [reply its/plugin/pom.xml]: Thanks, I will create a separate PR for this.

Comment (roberto-orlandi-sonarsource) [reply its/plugin/pom.xml]: It seems that it was decreased few minutes ago with this commit:
https://github.com/SonarSource/sonar-html/commit/0f76403603148d7a50037436b05bede658e24ba2

Comment (roberto-orlandi-sonarsource) [reply its/plugin/pom.xml]: Forget my last comment :D

**Review by roberto-orlandi-sonarsource — APPROVED** [review]
LGTM!

**Pull request merged**